docs/en/ingest-management/elastic-agent/configuration/providers/kubernetes_leaderelection-provider.asciidoc

```diff
@@ -23,8 +23,8 @@ providers.kubernetes_leaderelection:
   #leader_renewdeadline: 10
 ----
 
-`enabled`:: (Optional) Defaults to true. To explicitly disable the LeaderElection provider,
-set `enabled: false`.
```

Comment:
Don't change this.

It is still on, if its referenced. Setting to `enabled: false` would turn it off completely, even if it was referenced.